spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/network_manager_spec.rb

```diff
@@ -0,0 +1,37 @@
+describe ManageIQ::Providers::IbmCloud::PowerVirtualServers::NetworkManager do
+  context "ems" do
+    it "does not support network creation" do
+      ems = FactoryBot.create(:ems_ibm_cloud_power_virtual_servers_cloud)
+      expect(ems.supports_ems_network_new?).to eq(false)
+    end
+  end
+
+  context "singleton methods" do
+    it "returns the expected value for the description method" do
+      expect(described_class.description).to eq('IBM Cloud Networks')
+    end
+
+    it "returns the expected value for the ems_type method" do
+      expect(described_class.ems_type).to eq('ibm_cloud_network')
+    end
+
+    it "returns the expected value for the hostname_required? method" do
+      expect(described_class.hostname_required?).to eq(false)
+    end
+
+    it "returns the expected value for the default_blacklisted_event_names method" do
+      array = %w(
+        ConfigurationSnapshotDeliveryCompleted
+        ConfigurationSnapshotDeliveryStarted
+        ConfigurationSnapshotDeliveryFailed
+      )
+
+      expect(described_class.default_blacklisted_event_names).to eq(array)
+    end
+
+    it "returns the expected value for the display_name method" do
```

Comment:
I think we're just using the inherited `ManageIQ::Providers::NetworkManager` 'display_name' class method:
https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/network_manager.rb#L75-L77

Seems to me this test would belong in the specs for `ManageIQ::Providers::NetworkManager`:
https://github.com/ManageIQ/manageiq/blob/master/spec/models/manageiq/providers/network_manager_spec.rb